### DISCUSSION TITLE
Multi account support

### DISCUSSION THREAD

**Discussion by RomanMazeevRMR** [community/community#3337] [Enterprise]
Please add multi account support for users with different accounts. One for job and second personal.

**Comment by rezagh** [comment]
+1 on this.

**Reply by 9777410929** [reply]
hi

**Comment by candyho** [comment]
Hey there, we have just shipped a Beta for GitHub Enterprise Server 3.0 support on mobile recently which lets you switch between enterprise and personal accounts in-app. You need to be on GitHub Enterprise Server instance version 3.0 or greater, and install the latest version of our apps on your device . If you have met these requirements, you could sign in with a user account on GitHub.com, then long-press your Profile avatar at the bottom to add your Enterprise Account.

Hopefully we will be supporting more enterprise plans on mobile in the near future!

You can find more information about GitHub Enterprise Server 3.0 Support on mobile here - https://docs.github.com/en/enterprise-server@3.0/github/getting-started-with-github/github-for-mobile

**Reply by Airn5475** [reply]
When will the app get Enterprise Cloud support?

**Comment by hlsantos** [comment]
how about the ability to just logoff and login again with different account?  dont see logoff or sign out option.

**Reply by eliperkins** [reply]
@hlsantos to sign out of an account, you can navigate to Profile > Settings > Manage Accounts and either tap Edit to pick which account to sign out of, or swipe on a specific account to sign out of that account.

Alternatively, long-pressing on the Profile tab will bring up an account switcher popup with the same behavior.

Let me know how this works out for ya!

**Comment by rw10789** [comment]
> Please add multi account support for users with different accounts. One for job and second personal.

The new enterprise feature is great but it doesn't really help if you don't have an enterprise account for work.

I can login/logout of multiple accounts (and switch between) with all Google apps, Microsoft teams etc, Github should add that feature here too.

**Comment by DevAndArtist** [comment]
I can only second the request. I do have a dedicated account for work and one for my personal projects and I do not want to mix them on the same account. Many other platforms support multiple account usage (YouTube, Twitter, etc.). What's so special about it to support this highly requested feature on GitHub as well?

I for my part would love to be able to use the iOS contribution graph widget multiple times for several non-enterprise accounts.

**Comment by ohwhen-dd** [comment]
Bumping this one, would love to have this feature, same use case (personal and work account, both paid for). The lack of multi-user account support is actually causing me to use the app less.

**Comment by ghost** [comment]
'+' 1 on this.

**Comment by Angelk90** [comment]
@candyho , @amrehman: Will the ability to support more than one account (other than GitHub Enterprise Server) be implemented?

**Comment by dzmpr** [comment]
Waiting for this for a long time. +1

**Comment by Jerry23011** [comment]
I work with 2 accounts. +1

**Comment by datlechin** [comment]
+1

**Comment by thanhcsf** [comment]
Hello, Is there any update on this issue?

**Comment by aisgbnok** [comment]
You can definitely do this after today. [They added multiple account support in the latest app update!](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/)

***Note:** Since this is under the category **Mobile**, I'm only talking about the mobile app.*

**Comment by byrneh** [comment]
for browser access it is possible to be logged into.multiple accounts at the same time on a single device by using browser profiles and switching between these browser profiles

**Comment by eliperkins** [comment]
Hey @RomanMazeevRMR and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

This includes:
- Enterprise Cloud customers
- Enterprise Managed User customers
- Enterprise Server customers
- those power users out there who just happen to have separate personal and work accounts!

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.

**Reply by DevAndArtist** [reply]
Bug report: Doesn't work on the iPad. Still says `Manage Accounts` and `Add Enterprise Account` only. Works great on the iPhone though. Both apps are updated to the latest version with the new feature introduction changelogs.

**Reply by eliperkins** [reply]
@DevAndArtist Interesting! Would you be able to share the version and build number that you're using on your iPad? You should be able to find this in the footer in Settings within the app. I'll dig in to help debug this for ya.

Thanks for your help!

**Reply by DevAndArtist** [reply]
Sure thing. 
- version 1.125.0 (106453192)
- iPad Pro 11" 2018 with iOS 16.2

**Reply by DevAndArtist** [reply]
On the iPhone the version is different: 1.126.0 (107057637) 🤔

**Reply by DevAndArtist** [reply]
Okay for some reason, now the App Store shows `update` again. I swear I explicitly updated it today and then it said `open`. You can ignore it then. I finally updated and can add more accounts. 🎉 

Now it's 1.126.0 (107057637) on the iPad as well.

**Reply by eliperkins** [reply]
@DevAndArtist all good! I'll chalk this one ups to some App Store shenanigans. Thanks for diving in deeper here, and be sure to keep the feedback coming our way!

**Reply by andreinitescu** [reply]
Any plan for this feature any time soon for the web?

**Comment by noctyris** [comment]
# YES !
